code/datums/sexcon/sex_actions/toy/toy_other_anal.dm

```diff
@@ -19,11 +19,23 @@
 
 /datum/sex_action/toy_other_anal/on_start(mob/living/carbon/human/user, mob/living/carbon/human/target)
 	var/obj/item/dildo = get_dildo_in_either_hand(user)
-	user.visible_message(span_warning("[user] shoves \the [dildo] in [target]'s butt..."))
+	if(HAS_TRAIT(target, TRAIT_TINY) && !(HAS_TRAIT(user, TRAIT_TINY)))
```

Comment:
I don't think user's size should matter for this?

Reply:
It matters only because Seelie shrink items by using them - since normal humenlikes do not have this ability they will be shoving a full sized toy into a tiny Seelie.

Reply:
> Seelie shrink items by using them

Deep Seelie lore